splits/scl/README.md

```diff
@@ -0,0 +1,79 @@
+### Dataset description
+
+The SCL "six" variations set stems from a [2021 publication](https://academic.oup.com/bioinformaticsadvances/article/1/1/vbab035/6432029) (base on a [2017 publication](https://academic.oup.com/bioinformatics/article/33/21/3387/3931857)) and a [2022 publicaiton](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac278/6576357) in which the prediction of proteins subcellular localization where tested.
+
+The possible subcellular localizations are `Cytoplasm`, `Nucleus`, `Cell membrane`, `Mitochondrion`, `Endoplasmic reticulum`, `Lysosome/Vacuole`, `Golgi apparatus`, `Peroxisome`, `Extracellular` and `Plastid`.
+
+### Full dataset
+
+The `deeploc_our_train_set.fasta`, `deeploc_our_val_set.fasta`, `deeploc_our_test_set.fasta` and `setHARD.fast` are the source dataset from [2021 publication](https://academic.oup.com/bioinformaticsadvances/article/1/1/vbab035/6432029) used to create the DeepLoc 1.0 splits. The `human_sequences.tsv` is used to recover the human proteins from the DeepLoc 1.0 set.
+
+The `Swissprot_Train_Validation_dataset.csv` and the `hpa_testset.csv` are the training/validation and test set from the [2022 publicaiton](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkac278/6576357) used to create the DeepLoc 2.0 split.
+
+Due to the size of these files, they can be found at http://data.bioembeddings.com/public/FLIP/scl/.
+
+### Splits
+
+All splits are regression splits on the `Fittness` value reported by the [2016 publication](https://elifesciences.org/articles/16965) in the [supplement](https://doi.org/10.7554/eLife.16965.024).
```

Comment:
```suggestion
All splits are classification splits as explained prior.
```

Reply:
Oops, sorry, I copied an example to follow the structure and I didn't realize that I got out of this sentence.